### PR TITLE
Update/Edit AWX Docs

### PR DESCRIPTION
**Pull request by beeankha**
##### SUMMARY
Updating AWX docs to fix grammatical errors, improve comprehension and update outdated information (e.g., references to incorrect API endpoints).

##### ISSUE TYPE

 - Docs Pull Request

### CODE REVIEW

File: docs/notification_system.md [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
-A Notifier is an instance of a notification type (Email, Slack, Webhook, etc) with a name, description, and a defined configuration (A few examples: Username, password, server, recipients for the Email type.  Token and list of channels for Slack.  Url and Headers for webhooks)
+A Notifier is an instance of a notification type (Email, Slack, Webhook, etc.) with a name, description, and a defined configuration A few examples include:
 
-A Notification is a manifestation of the Notifier... for example, when a job fails a notification is sent using the configuration defined by the Notifier.
+* Username, password, server, recipients for the Email type.
+* Token and list of channels for Slack.
+* URL and Headers for webhooks.
 
-This PR implements the Notification system as outlined in the 3.0 Notifications spec.  At a high level the typical flow is:
+A Notification is a manifestation of the Notifier... for example, when a job fails, a notification is sent using the configuration defined by the Notifier.
```
Comment (ryanpetrello) [inline docs/notification_system.md]: We should change `Notifier` to `NotificationTemplate`, I think "notifier" nomenclature is from the _old_ implementation before @matburt rewrote this.

File: docs/notification_system.md [hunk 2]
```diff
@@ -1,40 +1,45 @@
...
 
-* User creates a Notifier at `/api/v1/notifiers`
-* User assigns the notifier to any of the various objects that support it (all variants of job templates as well as organizations and projects) and at the appropriate trigger level for which they want the notification (error, success, or any).   For example a user may wish to assign a particular Notifier to trigger when `Job Template 1` fails.   In which case they will associate the notifier with the job template at `/api/v1/job_templates/n/notifiers_error`.
+At a high level, the typical notification task flow is:
 
-## Notifier hierarchy
+* User creates a Notification at `/api/v2/notification_templates/`.
```
Comment (ryanpetrello) [inline docs/notification_system.md]: ...creates a `NotificationTemplate`...

File: docs/notification_system.md [hunk 3]
```diff
@@ -1,40 +1,45 @@
...
+* User assigns the notification to any of the various objects that support it (all variants of Job Templates as well as organizations and projects) and at the appropriate trigger level for which they want the notification (error, success, or any).   For example, a user may wish to assign a particular Notification to trigger when `Job Template 1` fails.   In which case they will associate the notification with the job template at `/api/v2/job_templates/n/notifiers_error`.
```
Comment (ryanpetrello) [inline docs/notification_system.md]: (NotificationTemplate)

Comment (ryanpetrello) [reply docs/notification_system.md]: > In which case they will associate the notification with the job template at `/api/v2/job_templates/n/notifiers_error`

This is out of date.  Let's either update or remove this sentence.

File: docs/notification_system.md (hunk 1 above)
Comment (ryanpetrello) [reply docs/notification_system.md]: This goes for most references to "notifier" in this doc.

File: docs/notification_system.md [hunk 4]
```diff
@@ -43,179 +48,188 @@ The currently defined Notification Types are:
...
+* Verify single and multiple recipients.
 * Verify message subject and contents are formatted sanely.  They should be plaintext but readable.
 
 ### Test Service
 
-Either setup a local smtp mail service here are some options:
+Set up a local SMTP mail service.  Some options are listed below:
+
+* Postfix service on galaxy: https://galaxy.ansible.com/debops/postfix/
+* Mailtrap has a good free plan that should provide all of the features necessary: https://mailtrap.io/
```
Comment (ryanpetrello) [inline docs/notification_system.md:70]: A docker container is another good option here:

`docker run --network="tools_default" -p 25:25 -e maildomain=mail.example.com -e smtp_user=user:pwd --name postfix -d catatnight/postfix`

File: docs/notification_system.md [hunk 5]
```diff
@@ -43,179 +48,188 @@ The currently defined Notification Types are:
...
 
 The following should be performed for good acceptance:
 
-* Test single and multiple channels and good formatting of the message.   Note that slack notifications only contain the minimal information
+* Test single and multiple channels and good formatting of the message. Note that slack notifications only contain the minimal information.
 
 ### Test Service
 
-Any user of the Ansible slack service can create a bot integration (which is how this notification is implemented).   Remember to invite the bot to the channel first.
+Any user of the Ansible Slack service can create a bot integration (which is how this notification is implemented). Remember to invite the bot to the channel first.
```
Comment (ryanpetrello) [inline docs/notification_system.md]: We should remove this entire section about the Ansible Slack service - it's not public-facing.

File: docs/notification_system.md [hunk 6]
```diff
@@ -43,179 +48,188 @@ The currently defined Notification Types are:
...
 
-## Hipchat
 
-There are several ways to integrate with hipchat.  The Tower implementation uses Hipchat "Integrations".  Currently you can find this at the bottom right of the main hipchat webview.  From there you will select "Build your own Integration".   After creating that it will list the `auth_token` that needs to be supplied to Tower.  Some other relevant details on the fields accepted by Tower for the Hipchat notification type:
+## HipChat
```
Comment (ryanpetrello) [inline docs/notification_system.md]: I'm fine if we go ahead and remove this entire section - HipChat is dead:

> On July 26, 2018, Atlassian announced that HipChat and Stride would be discontinued February 15, 2019, and that it had reached a deal to sell their intellectual property to Slack.

**Review by ryanpetrello — APPROVED** [review]
Thanks for cleaning this up, left a few minor comments.

File: docs/inventory_plugins.md [hunk 7]
```diff
@@ -1,153 +1,79 @@
...
-More hostvars will appear if the inventory plugins are used.
-To maintain backward compatibility,
-the old names are added back where they have the same meaning as a
-variable returned by the plugin. New names are not removed.
+More hostvars will appear if the inventory plugins are used. To maintain backward compatibility, the old names are added back where they have the same meaning as a variable returned by the plugin. New names are not removed.
 
 A small number of hostvars will be lost because of general deprecation needs.
 
-#### Host names
+#### Host nNames
```
Comment (rooftopcellist) [inline docs/inventory_plugins.md]: `nNames` --> `Names`

File: docs/inventory_plugins.md [hunk 8]
```diff
@@ -1,153 +1,79 @@
...
+The `inputs` for this credential constitute one source of data for running inventory updates. The following fields from the `InventoryUpdate` model are also data sources, including:
 
  - `source_vars`
  - `source_regions`
  - `instance_filters`
  - `group_by`
 
-The way these data are applied to the environment (including files and 
-environment vars) is highly dependent on the specific source.
+The way these data are applied to the environment (including files and environment vars) is highly dependent on the specific source.
```
Comment (rooftopcellist) [inline docs/inventory_plugins.md]: "The way this data is applied"  sounds much better to me, but maybe there is some grammar technicality I don't know about here..

File: docs/notification_system.md [hunk 9]
```diff
@@ -1,40 +1,43 @@
...
-Notifiers assigned at certain levels will inherit notifiers defined on parent objects as such:
+Notification templates assigned at certain levels will inherit notifications defined on parent objects as such:
 
-* Job Templates will use notifiers defined on it as well as inheriting notifiers from the Project used by the Job Template and from the Organization that it is listed under (via the Project).
-* Project Updates will use notifiers defined on the project and will inherit notifiers from the Organization associated with it.
-* Inventory Updates will use notifiers defined on the Organization that it is listed under
-* Ad-hoc commands will use notifiers defined on the Organization that the inventory is associated with
+* Job Templates will use notifications defined on it as well as inheriting notifications from the Project used by the Job Template and from the Organization that it is listed under (via the Project).
+* Project Updates will use notifications defined on the project and will inherit notifications from the Organization associated with it.
+* Inventory Updates will use notifications defined on the Organization under which it is listed.
```
Comment (rooftopcellist) [inline docs/notification_system.md]: "defined on the Organization it is in."  -- maybe conveys it more succintly

File: docs/inventory_plugins.md (hunk 8 above)
Comment (beeankha) [reply docs/inventory_plugins.md]: Your suggestion sounds a lot better, editing this.

**Pull request merged**